docs/source/conf.py

```diff
@@ -98,6 +99,12 @@
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
+# Options for Sphinx copybutton extension
+# ---------------------------------------
+
+# Matches prompts - "$ ", ">>>" and "..."
+copybutton_prompt_text = r">>> |\.\.\. |\$ "
```

Comment:
I feel like ellipses might be used in places which aren't code prompts. Or am I misunderstanding how this works?

Reply:
Triple ellipses `...`?

The intention here is to support codeblocks that look like

```
>>> CustomClass(
...    var_a=1,
...    var_b=2,
... )
```

Reply:
Ahh OK. That makes more sense.

Reply:
Thanks John! Merging!